### PR TITLE
Developer guide touchup

### PR DESCRIPTION
**Pull request by XXJJXJ**
This PR changes UG(super minor tweak) and DG

I omited list and listEvents, cuz they are very trivial

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -985,15 +1012,73 @@ testers are expected to do more *exploratory* testing.
...
+   1. Test case: `addEvent n/Graduation d/2024-05-25 t/1200`<br>
+      Expected: Event is added to the event list. Details of the event shown in the status message.
 
 ###  Deleting an event
 
-1. Deleting an event while all events are being shown
+1. Delete an event while all events are being shown
 
-   1. Prerequisites: List all events using the `listEvents` command. Multiple events in the list.
+   1. Prerequisites: List all events using the `listEvents` command. Multiple events in the event list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1078]:
```suggestion
   1. Prerequisites: All events listed using the `listEvents` command. Multiple events in the event list.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -985,15 +1012,73 @@ testers are expected to do more *exploratory* testing.
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
+### Editing a participant
+
+1. Edit a participant's name to another non-existent participant
```
Comment (ntwbruce) [inline docs/DeveloperGuide.md:1017]:
```suggestion
1. Edit a participant to one that does not already exist
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -985,15 +1012,73 @@ testers are expected to do more *exploratory* testing.
...
+      Expected: Details of the first participant in the participant list shown in the status message.
+
+### Adding a Next-of-Kin to a participant
+
+1. Add a Next-of-Kin(NOK) to a participant
+
+   1. Prerequisite: At least 1 participant in the participant list.
+   
+   2. Test case: `addNok 1 n/Patrick Yeoh p/91234444 tag/Cousin`<br>
+      Expected: NOK added to the first participant in the list. Details of action shown in the status message.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1051]:
```suggestion
      Expected: NOK is added to the first participant in the list. Details of action shown in the status message.
```

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -985,15 +1012,73 @@ testers are expected to do more *exploratory* testing.
...
+
+   2. Test case: `edit 1 n/Alexis Peoh`<br>
+      Expected: Participant at index 1 renamed to "Alexis Peoh" with all other details remained the same.
+
+
+### Locating participants by name
+
+1. Find a participant with name "David"
+
+   1. Prerequisite: List all participants using the `list` command. Multiple participants in the participant list.
```
Comment (ntwbruce) [inline docs/DeveloperGuide.md:1029]:
```suggestion
   1. Prerequisite: List all participants using the `list` command. At least 1 participant in the participant list.
```

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
       list size)<br>
       Expected: Similar to previous.
 
+### Editing an event
+
+1. Edit an event to a non-existent event
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1092]:
```suggestion
1. Edit an event (new name and date of the event does not match any other event's)
```

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -985,15 +1012,73 @@ testers are expected to do more *exploratory* testing.
...
+   1. Prerequisite: At least 1 participant in the participant list.
+   
+   2. Test case: `addNok 1 n/Patrick Yeoh p/91234444 tag/Cousin`<br>
+      Expected: NOK added to the first participant in the list. Details of action shown in the status message.
+
+### Deleting a Next-of-Kin of a participant
+
+1. Delete a NOK from a participant
+
+   1. Prerequisite: At least 1 participant and this participant has at least 1 NOK.
```
Comment (ntwbruce) [inline docs/DeveloperGuide.md:1057]:
```suggestion
   1. Prerequisite: At least 1 participant in the participant list, and the first participant has at least 1 NOK.
```

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
       Expected: Similar to previous.
 
+### Editing an event
+
+1. Edit an event to a non-existent event
+
+   1. Prerequisite: At least 1 event in event list. No events with the same name and date as the new edited event.
+
+   2. Test case: `editEvent 1 n/CS2100 finals d/2021-11-20`<br>
+      Expected: Event name and date is edited, time remains the same. Details of new event shown in status message.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1097]:
```suggestion
      Expected: Event name and date are edited, time remains the same. Details of new event shown in status message.
```

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+
+### Marking event as done
+
+1. Mark an event as done
+
+   1. Prerequisite: At least 1 event in event list. Event must be "Uncompleted".
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1103]:
```suggestion
   1. Prerequisite: At least one event in event list. Event must be "Uncompleted".
```

File: docs/DeveloperGuide.md [hunk 9]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+
+   2. Test case: `done 1`<br>
+      Expected: First event in event list marked as done. Details of completed event shown in status message.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1106]:
```suggestion
      Expected: First event in event list is marked as done. Details of completed event shown in status message.
```

File: docs/DeveloperGuide.md [hunk 10]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+
+### Locating events by name
+
+1. Find events by event names
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1110]:
```suggestion
1. Find events by event name
```

File: docs/DeveloperGuide.md [hunk 11]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+
+   1. Prerequisite: List all events using the `listEvents` command. Multiple events in the event list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1112]:
```suggestion
   1. Prerequisite: All events listed using the `listEvents` command. At least one event in the event list.
```

File: docs/DeveloperGuide.md (hunk 5 above)
Comment (ntwbruce) [inline docs/DeveloperGuide.md:1092]:
```suggestion
1. Edit an event to one that does not already exist
```

File: docs/DeveloperGuide.md [hunk 12]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+
+   2. Test case: `findEvent Monthly`<br>
+      Expected: Event list filtered to show only events with "Monthly" in their names. Could be empty if
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1115]:
```suggestion
      Expected: Event list is filtered to show only events with "Monthly" in their names. The list may be empty if
```

File: docs/DeveloperGuide.md [hunk 13]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+      no event has "Monthly" in their name.
+
+### Filtering events by time
+
+1. Filter by date only
+
+   1. Prerequisite: List all events using the `listEvents` command. Multiple events in the event list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1122]:
```suggestion
   1. Prerequisite: All events listed using the `listEvents` command. At least one event in the event list.
```

File: docs/DeveloperGuide.md [hunk 14]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+   2. Test case: `filterEvents d/2021-09-30`<br>
+      Expected: Events occurring on 30th September 2021 will be shown in the event list. Could be empty if no event
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1124]:
```suggestion
      Expected: Events occurring on 30th September 2021 will be shown in the event list. The list may be empty if no event
```

File: docs/DeveloperGuide.md [hunk 15]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+      occurs on this date.
+
+2. Filter by date and time
+
+   1. Prerequisite: List all events using the `listEvents` command. Multiple events in the event list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1129]:
```suggestion
   1. Prerequisite: All events listed using the `listEvents` command. At least one event in the event list.
```

File: docs/DeveloperGuide.md [hunk 16]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+   2. Test case: `filterEvents d/2021-09-30 t/1200`<br>
+      Expected: Events occurring on 30th September 2021 at 1200 will be shown in the event list. Could be empty if
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1131]:
```suggestion
      Expected: Events occurring on 30th September 2021 at 1200 are be shown in the event list. The list may be empty if
```

File: docs/DeveloperGuide.md [hunk 17]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+      no event occurs on this date and time.
+
+### Sorting events by time
+
+1. Sort the event list in chronological order
+
+   1. Prerequisite: List all events using the `listEvents` command. Multiple events in the event list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1138]:
```suggestion
   1. Prerequisite: All events listed using the `listEvents` command. At least one event in the event list.
```

File: docs/DeveloperGuide.md [hunk 18]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+   2. Test case: `sortEvents`<br>
+      Expected: Events will be sorted chronologically. For events with the same date, those without time will be on top
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1140]:
```suggestion
      Expected: Events will be sorted chronologically. For events with the same date, events without time will be placed higher than events with time specified.
```

File: docs/DeveloperGuide.md [hunk 19]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+      of those with time.
+
+### Showing an event’s details
+
+1. Show the details of an event
+
+   1. Prerequisite: List all events using the `listEvents` command. At least 1 event in the event list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1147]:
```suggestion
   1. Prerequisite: All events listed using the `listEvents` command. At least one event in the event list.
```

File: docs/DeveloperGuide.md (hunk 11 above)
Comment (ntwbruce) [inline docs/DeveloperGuide.md:1112]:
```suggestion
   1. Prerequisite: List all events using the `listEvents` command. At least 1 event in the event list.
```

File: docs/DeveloperGuide.md [hunk 20]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+   2. Test case: `showDetails 1`<br>
+      Expected: Details of the first event in the event list is shown in the status message.
+
+### Adding a participant to an event
+
+1. Enroll a participant into an event
+
+   1. Prerequisite: List all events and participants using `listEvents` and `list` commands. At least 1 event and 1 participant
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1155]:
```suggestion
   1. Prerequisite: All events and participants listed using `listEvents` and `list` commands respectively. At least one event and one participant
```

File: docs/DeveloperGuide.md [hunk 21]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+      in the event and participant list respectively. Participant should not be already enrolled in the event.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1156]:
```suggestion
      in the event and participant lists respectively. The participant should not be already enrolled in the event.
```

File: docs/DeveloperGuide.md [hunk 22]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+   2. Test case: `enroll 1 1`<br>
+      Expected: First participant in the participant list enrolled into the first event in the event list. Details of action
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1158]:
```suggestion
      Expected: First participant in the participant list is enrolled into the first event in the event list. Details of action
```

File: docs/DeveloperGuide.md (hunk 16 above)
Comment (ntwbruce) [inline docs/DeveloperGuide.md:1131]:
```suggestion
      Expected: Events occurring on 30th September 2021 at 12:00pm will be shown in the event list. Could be empty if
```

File: docs/DeveloperGuide.md [hunk 23]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+      shown in the status message.
+
+### Removing a participant from an event
+
+1. Expel a participant from an event
+
+   1. Prerequisite: List all events and participants using `listEvents` and `list` commands. At least 1 event and 1 participant
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1165]:
```suggestion
   1. Prerequisite: All events and participants listed using `listEvents` and `list` commands respectively. At least one event and one participant
```

File: docs/DeveloperGuide.md [hunk 24]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+      in the event and participant list respectively. Participant should be already enrolled in the event.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1166]:
```suggestion
      in the event and participant lists respectively. The participant should be already enrolled in the event.
```

File: docs/DeveloperGuide.md [hunk 25]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+   2. Test case: `expel 1 1`<br>
+      Expected: First participant in the participant list expelled from the first event in the event list. Details of action
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1168]:
```suggestion
      Expected: First participant in the participant list is expelled from the first event in the event list. Details of action
```

File: docs/DeveloperGuide.md [hunk 26]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+      shown in the status message.
+
+### Showing an event’s participants
+
+1. Show a list of participants in an event
 
+   1. Prerequisite: List all events using `listEvents` command. At least 1 event in the event list. There should be at least
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1175]:
```suggestion
   1. Prerequisite: All events listed using the `listEvents` command. At least one event in the event list. There should be at least
```

File: docs/DeveloperGuide.md [hunk 27]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+      1 participant enrolled in the event.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1176]:
```suggestion
      one participant enrolled in the event.
```

File: docs/DeveloperGuide.md [hunk 28]
```diff
@@ -1002,7 +1087,95 @@ testers are expected to do more *exploratory* testing.
...
+
+   1. Prerequisite: List all events and participants using `listEvents` and `list` commands. At least 1 event and 1 participant
+      in the event and participant list respectively. Participant should be already enrolled in the event.
+   2. Test case: `expel 1 1`<br>
+      Expected: First participant in the participant list expelled from the first event in the event list. Details of action
+      shown in the status message.
+
+### Showing an event’s participants
+
+1. Show a list of participants in an event
```
Comment (ntwbruce) [inline docs/DeveloperGuide.md:1173]:
```suggestion
1. Show a list of an event's participants
```

**Review by ntwbruce — APPROVED** [review]
Just some minor phrasing edits, otherwise LGTM

**Review by lowkangn — CHANGES_REQUESTED** [review]
Some more consistency fixes 😷 Sorry for being too uptight

File: docs/DeveloperGuide.md [hunk 29]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
    1. Test case: `delete 0`<br>
-      Expected: No participant is deleted. Error details shown in the status message.
+      Expected: No participant is deleted. Error details are shown in the status message.
 
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
+### Editing a participant
+
+1. Edit a participant's name (new name is unique and does not match any other participant's)
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1017]:
```suggestion
1. Edit a participant's name
```

File: docs/DeveloperGuide.md [hunk 30]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+   1. Prerequisite: All participants listed using the `list` command. At least one participant is in the participant list.
+
+   2. Test case: `view 1`<br>
+      Expected: Details of the first participant in the participant list are shown in the status message.
+
+### Adding a Next-of-Kin to a participant
+
+1. Add a Next-of-Kin(NOK) to a participant
+
+   1. Prerequisite: At least one participant in the participant list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1048]:
```suggestion
   1. Prerequisite: At least one participant is in the participant list.
```
```suggestion
   1. Prerequisite: At least one participant in the participant list.
```

File: docs/DeveloperGuide.md [hunk 31]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+   
+   2. Test case: `addNok 1 n/Patrick Yeoh p/91234444 tag/Cousin`<br>
+      Expected: The NOK is added to the first participant in the list. Details of the action are shown in the status message.
+
+### Deleting a Next-of-Kin of a participant
+
+1. Delete a NOK from a participant
+
+   1. Prerequisite: At least 1 participant in the participant list, and the first participant has at least 1 NOK.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1057]:
```suggestion
   1. Prerequisite: At least one participant is in the participant list, and the first participant has at least one NOK.
```

File: docs/DeveloperGuide.md [hunk 32]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+   1. Test case: `addEvent n/Graduation d/2024-05-25 t/1200`<br>
+      Expected: The event is added to the event list. Details of the event are shown in the status message.
 
 ###  Deleting an event
 
-1. Deleting an event while all events are being shown
+1. Delete an event while all events are shown
 
-   1. Prerequisites: List all events using the `listEvents` command. Multiple events in the list.
+   1. Prerequisites: All events listed using the `listEvents` command. At least one event in the event list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1078]:
```suggestion
   1. Prerequisites: All events listed using the `listEvents` command. At least one event is in the event list.
```

File: docs/DeveloperGuide.md [hunk 33]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
 
    4. Other incorrect deleteEvent commands to try: `deleteEvent`, `deleteEvent x`, `...` (where x is larger than the 
       list size)<br>
       Expected: Similar to previous.
 
+### Editing an event
+
+1. Edit an event (new name and date of the event does not match any other event's)
+
+   1. Prerequisite: At least one event in the event list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1094]:
```suggestion
   1. Prerequisite: At least one event is in the event list. No other events are named "CS2100 finals" **and** fall on 20th November 2021.
```

File: docs/DeveloperGuide.md [hunk 34]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+
+   2. Test case: `editEvent 1 n/CS2100 finals d/2021-11-20`<br>
+      Expected: The name and date of the event are edited, but time remains the same. Details of the new event are shown in the status message.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1097]:
```suggestion
      Expected: The name and date of the event are edited, but time is unchanged. Details of the new event are shown in the status message.
```

File: docs/DeveloperGuide.md [hunk 35]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+
+### Marking event as done
+
+1. Mark an event as done
+
+   1. Prerequisite: At least one event in event list. The first event must be "Uncompleted".
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1103]:
```suggestion
   1. Prerequisite: At least one event is in event list. The first event must be "Uncompleted".
```

File: docs/DeveloperGuide.md [hunk 36]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
-      Expected: No event is deleted. Error details shown in the status message.
+      Expected: No event is deleted. Error details are shown in the status message.
 
    4. Other incorrect deleteEvent commands to try: `deleteEvent`, `deleteEvent x`, `...` (where x is larger than the 
       list size)<br>
       Expected: Similar to previous.
 
+### Editing an event
+
+1. Edit an event (new name and date of the event does not match any other event's)
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1092]:
```suggestion
1. Edit an event
```

File: docs/DeveloperGuide.md [hunk 37]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+      Expected: No participant is deleted. Error details are shown in the status message.
 
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
+### Editing a participant
+
+1. Edit a participant's name (new name is unique and does not match any other participant's)
+
+   1. Prerequisite: At least one participant is in the participant list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1019]:
```suggestion
   1. Prerequisite: At least one participant is in the participant list. No other participants are named "Alexis Peoh".
```

File: docs/DeveloperGuide.md [hunk 38]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+   1. Prerequisite: At least one event in event list. The first event must be "Uncompleted".
+
+   2. Test case: `done 1`<br>
+      Expected: The first event in the event list is marked as done. Details of the completed event are shown in the status message.
+
+### Locating events by name
+
+1. Find events by event name
+
+   1. Prerequisite: All events listed using the `listEvents` command. At least one event in the event list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1112]:
```suggestion
   1. Prerequisite: All events listed using the `listEvents` command. At least one event is in the event list.
```

File: docs/DeveloperGuide.md [hunk 39]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+
+   2. Test case: `findEvent Monthly`<br>
+      Expected: Event list is filtered to show only events with "Monthly" in their names. The list may be empty if
+      no event has "Monthly" in their name.
+
+### Filtering events by time
+
+1. Filter by date only
+
+   1. Prerequisite: All events listed using the `listEvents` command. At least one event in the event list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1122]:
```suggestion
   1. Prerequisite: All events listed using the `listEvents` command. At least one event is in the event list.
```

File: docs/DeveloperGuide.md [hunk 40]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+   2. Test case: `filterEvents d/2021-09-30`<br>
+      Expected: Events occurring on 30th September 2021 will be shown in the event list. The list may be empty if no event
+      occurs on this date.
+
+2. Filter by date and time
+
+   1. Prerequisite: All events listed using the `listEvents` command. At least one event in the event list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1129]:
```suggestion
   1. Prerequisite: All events listed using the `listEvents` command. At least one event is in the event list.
```

File: docs/DeveloperGuide.md [hunk 41]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+   2. Test case: `filterEvents d/2021-09-30 t/1200`<br>
+      Expected: Events occurring on 30th September 2021 at 12:00pm will be shown in the event list. Could be empty if
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1131]:
```suggestion
      Expected: Events occurring on 30th September 2021 at 12:00pm will be shown in the event list. The list may be empty if
```

File: docs/DeveloperGuide.md [hunk 42]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+      no event occurs on this date and time.
+
+### Sorting events by time
+
+1. Sort the event list in chronological order
+
+   1. Prerequisite: All events listed using the `listEvents` command. At least one event in the event list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1138]:
```suggestion
   1. Prerequisite: All events listed using the `listEvents` command. At least one event is in the event list.
```

File: docs/DeveloperGuide.md [hunk 43]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+   2. Test case: `sortEvents`<br>
+      Expected: The events in the event list will be sorted chronologically. For events with the same date, events without time will be placed higher than events with time specified.
+      of those with time.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1141]:
```suggestion
```

File: docs/DeveloperGuide.md [hunk 44]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+
+### Showing an event’s details
+
+1. Show the details of an event
+
+   1. Prerequisite: All events listed using the `listEvents` command. At least one event in the event list.
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1147]:
```suggestion
   1. Prerequisite: All events listed using the `listEvents` command. At least one event is in the event list.
```

File: docs/DeveloperGuide.md [hunk 45]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+   2. Test case: `showDetails 1`<br>
+      Expected: Details of the first event in the event list are shown in the status message.
+
+### Adding a participant to an event
+
+1. Enroll a participant into an event
+
+   1. Prerequisite: All events and participants listed using `listEvents` and `list` commands respectively. At least one event and one participant
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1155]:
```suggestion
   1. Prerequisite: All events and participants listed using `listEvents` and `list` commands respectively. At least one event and one participant are
```

File: docs/DeveloperGuide.md [hunk 46]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+      in the event and participant lists respectively. The participant should not be already enrolled in the event.
+   2. Test case: `enroll 1 1`<br>
+      Expected: The first participant in the participant list is enrolled into the first event in the event list. Details of the action
+      are shown in the status message.
+
+### Removing a participant from an event
+
+1. Expel a participant from an event
+
+   1. Prerequisite: All events and participants listed using `listEvents` and `list` commands respectively. At least one event and one participant
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1165]:
```suggestion
   1. Prerequisite: All events and participants listed using `listEvents` and `list` commands respectively. At least one event and one participant are
```

File: docs/DeveloperGuide.md [hunk 47]
```diff
@@ -968,41 +984,198 @@ testers are expected to do more *exploratory* testing.
...
+      in the event and participant lists respectively. The participant should be already enrolled in the event.
+   2. Test case: `expel 1 1`<br>
+      Expected: The first participant in the participant list is expelled from the first event in the event list. Details of the action
+      are shown in the status message.
+
+### Showing an event’s participants
+
+1. Show a list of an event's participants
 
+   1. Prerequisite: All events listed using the `listEvents` command. At least one event in the event list. There should be at least
```
Comment (lowkangn) [inline docs/DeveloperGuide.md:1175]:
```suggestion
   1. Prerequisite: All events listed using the `listEvents` command. At least one event is in the event list. There should be at least
```

**Review by lowkangn — APPROVED** [review]
Just a few more edits and should be good to go

**Pull request merged**